### PR TITLE
revert PR 184, PR 190 and PR 182

### PR DESCRIPTION
**Pull request by rahulait**
### General:

* [x] Have you removed all sensitive information, including but not limited to access keys and passwords?
* [x] Have you checked to ensure there aren't other open or closed [Pull Requests](../../pulls) for the same bug/feature/question?

### Pull Request Guidelines:

1. [ ] Does your submission pass tests?
1. [ ] Have you added tests? 
1. [ ] Are you addressing a single feature in this PR? 
1. [ ] Are your commits atomic, addressing one change per commit?
1. [ ] Are you following the conventions of the language? 
1. [ ] Have you saved your large formatting changes for a different PR, so we can focus on your work?
1. [ ] Have you explained your rationale for why this feature is needed? 
1. [ ] Have you linked your PR to an [open issue](https://blog.github.com/2013-05-14-closing-issues-via-pull-requests/)

### CODE REVIEW

**Review by tchinmai7 — APPROVED** [review]
tested manually that it works - will wait for @luthermonson's confirmation

File: deploy/chart/templates/daemonset.yaml [hunk 1]
```diff
@@ -33,18 +33,6 @@ spec:
             - --v=3
             - --port=0
             - --secure-port=10253
-            {{- if .Values.linodegoDebug }}
```
Comment (rosskirkpat) [inline deploy/chart/templates/daemonset.yaml:36]: The `linodego-debug` flag is still present, we should ensure that the helm chart still supports enabling it post-revert.

**Review by rosskirkpat — CHANGES_REQUESTED** [review]
The node cache TTL is configurable via an envvar (`LINODE_INSTANCE_CACHE_TTL`), however, the helm chart does not offer a method of setting that variable during deployment. 

I believe this should be added to allow fine tuning CCM performance as well as aiding in future debugging/troubleshooting.

**Comment by rahulait** [comment]
> The node cache TTL is configurable via an envvar (`LINODE_INSTANCE_CACHE_TTL`), however, the helm chart does not offer a method of setting that variable during deployment.
> 
> I believe this should be added to allow fine tuning CCM performance as well as aiding in future debugging/troubleshooting.

I think we can do that by setting additional env vars here:
https://github.com/linode/linode-cloud-controller-manager/blob/main/deploy/chart/values.yaml#L53-L58

**Review by rosskirkpat — APPROVED** [review]

**Review by luthermonson — APPROVED** [review]

**Pull request merged**